jukebox/hparams.py

```diff
@@ -170,8 +172,10 @@ def setup_hparams(hparam_set_names, kwargs):
     attn_order=12,
     blocks=64,
     init_scale=0.2,
+    c_res=1,
```

Comment:
Would this affect sampling?

Reply:
good q, checkpointing is set for training